src/sentry/search/events/filter.py

```diff
@@ -550,6 +559,8 @@ def parse_semver(version, operator) -> Optional[SemverFilter]:
                 0 if prerelease else 1,
                 prerelease,
             ],
+            [],
```

Comment:
Why do we pass this list here? If you want leave this as optional, you can pass `negated` as a kwarg.